dotcom-rendering/src/server/htmlPageTemplate.ts

```diff
@@ -391,7 +392,7 @@ https://workforus.theguardian.com/careers/product-engineering/
 						: ''
 				}
                 ${scriptTags.join('\n')}
-                <style class="webfont">${fontsCss}</style>
+				<style class="webfont">${new CleanCSS().minify(rawFontsCss).styles}</style>
```

Comment:
As this is inside the `htmlPageTemplate` function, I think this means the minification will now run on every request, whereas previously the result was generated once on startup and then cached in a variable? Can we likewise pull the result of this out into a variable in this file and reuse it?